src/user/top_artists.rs

```diff
@@ -59,6 +59,7 @@ impl TopArtists {
 }
 
 /// Allows users to specify the period of which they'd like to retrieve top artist data for.
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq, PartialOrd, Ord)]
```

Comment:
Little curiosity question regarding PartialOrd and Ord: the current definition order puts Overall before SevenDays, making the order somewhat logically mixed.

Shouldn't either
a) definition order be shuffled for shortest-to-longest order?
or b) PartialOrd, Ord **not** be derived?

Reply:
Yes! Thanks for catching that, I'll update the field order.